tests/testnet-calls.py

```diff
@@ -0,0 +1,343 @@
+from web3 import Web3
+from web3.exceptions import ContractCustomError, ContractLogicError
+import json
+from eth_account.messages import encode_defunct
+import time
+import sys
+import os
+import random
+from eip712.messages import EIP712Message
+from helpers import (
+    TEST_NET_VARS,
+    check_subgraph_signer,
+    check_deployed_subgraph_transaction
+)
+test_net_type = sys.argv[1]
+testnet = "eth-goerli"
+if(test_net_type != 'eth'):
+    testnet = "aribitrum-goerli"
+broadcast = False
+AMOUNT_TO_REDEEM = 5
+SUBGRAPH_ENDPOINT=TEST_NET_VARS[testnet]['subgraph-endpoint']
+STAKING_ADDRESS=Web3.to_checksum_address(TEST_NET_VARS[testnet]['staking'])
+GRAPH_TOKEN=Web3.to_checksum_address(TEST_NET_VARS[testnet]['graphToken'])
+ESCROW_ADDRESS=Web3.to_checksum_address(TEST_NET_VARS[testnet]['escrow'])
+TAP_ADDRESS=Web3.to_checksum_address(TEST_NET_VARS[testnet]['tapVerifier'])
+ALLOCATIONIDTRACKER=Web3.to_checksum_address(TEST_NET_VARS[testnet]['allocationIDTracker'])
+CHAIN_ID = TEST_NET_VARS[testnet]['chainID']
+w3 = Web3(Web3.WebsocketProvider(TEST_NET_VARS[testnet]['eth_rpc']))
+w3.eth.account.enable_unaudited_hdwallet_features()
+INDEXER_MNEMONIC = sys.argv[2]
+TAPDeployer = w3.eth.account.from_mnemonic(INDEXER_MNEMONIC, account_path="m/44'/60'/0'/0/1")
+INDEXER_ACCOUNT = w3.eth.account.from_mnemonic(INDEXER_MNEMONIC, account_path="m/44'/60'/0'/0/0")
+INDEXER_PK = INDEXER_ACCOUNT.key
+INDEXER = INDEXER_ACCOUNT.address
```

Comment:
Are these 2 really needed? Why not use `INDEXER_ACCOUNT.key` and `INDEXER_ACCOUNT.address` wherever needed?
Also, `INDEXER` should be `INDEXER_ADDRESS` for consistency.